frontend/src/features/admin-form/settings/SettingsEmailNotificationsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import { Skeleton, Stack, Wrap } from '@chakra-ui/react'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EmailFormSection } from './components/EmailFormSection'
+import {
+  isEmailMode,
+  isEmailOrStorageMode,
+  useAdminFormSettings,
+} from './queries'
+
+export const EmailSection = (): JSX.Element => {
+  const { data: settings, isLoading: isLoadingSettings } =
+    useAdminFormSettings()
+
+  return (
+    <Skeleton isLoaded={!isLoadingSettings && !!settings}>
+      <Stack spacing="2rem">
+        {settings && isEmailOrStorageMode(settings) ? (
+          <EmailFormSection
+            emails={settings.emails}
+            isRequired={isEmailMode(settings)}
+          />
+        ) : null}
```

Comment:
Good practice of using ternaries (`P ? A : null`) instead of short circuiting (`P && A`), which should help in mitigating hard to find bugs.